### DISCUSSION TITLE
Possible to get gtag client_id and session_id for manual cross domain tracking?

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [gtm-support/vue-gtm#473] [General]
Hello,
With this module is it possible to get gtag client_id and session_id for manual cross domain tracking?

such as: https://support.google.com/analytics/answer/10071811?hl=en#zippy=%2Cmanual-setup

**Comment by Shinigami92** [comment]
tldr: no
reason: this is google tag manager, not ga4 support
